src/components/radios/index.njk

```diff
@@ -44,6 +44,11 @@ Use this varation by wrapping the text field in a div with class `radio__other`,
     patternlibExample({"path": "components/radios/examples/radios-with-or/index.njk"})
 }}
 
+## Radios with with the option to clear
```

Comment:
```suggestion
## Radios with the option to clear
```

Reply:
...maybe include '...and always visible 'other' input'

Reply:
Yeah I'll be updating the docs fully.